addons/html_editor/static/src/core/selection_plugin.js

```diff
@@ -125,7 +125,7 @@ export class SelectionPlugin extends Plugin {
                     let [anchorNode, anchorOffset] = startPos(container);
                     if (
                         anchorNode.firstChild &&
-                        anchorNode.firstChild.getAttribute("contenteditable") === "false"
+                        anchorNode.firstChild.isContentEditable === false
```

Comment:
nitpicking maybe `!anchorNode.firstChild.isContentEditable`, but doesn't really matter.

Also one of the enterprise tests failed so I rebuilt that test